### ISSUE TITLE
Shoulder Hawkins test

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11001]
**Preferred term label:**
Shoulder Hawkins test

**Synonyms**

**Definition (free text, please give PubMed ID)**
Manual internal rotation on the forearm with the shoulders flexed forward to 90 degrees, elbows bent to 90 degrees so that the forearm is parallel to the ground, and palm facing down recreates pain in the shoulder region; named after Dr. Hawkins; also known as Hawkins impingement test; also known as Kennedy impingement test

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Shoulder external impingement syndrome, Distal clavicle osteolysis (DCO), Rotator cuff tear, Rotator cuff tendonitis

**Your nano-attribution (ORCID)**
https://www.physio-pedia.com/Hawkins_/_Kennedy_Impingement_Test_of_the_Shoulder

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001372
- label: Positive Hawkins test
- parents: [Positive provocative functional shoulder test (HP:0025823)](https://hpo.jax.org/app/browse/term/HP:0025823)
- pmid: PMID:10979525
- def: Manual internal rotation on the forearm with the shoulders flexed forward to 90 degrees, elbows bent to 90 degrees so that the forearm is parallel to the ground, and palm facing down recreates pain in the shoulder region; named after Dr. Hawkins.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11001
- synonyms: Positive Kennedy impingement test(NONE)|Positive Hawkins impingement test(NONE)
